app/templates/index.html

```diff
@@ -39,79 +38,20 @@
       {% include "toolkit/browse-list.html" %}
     {% endwith %}
   </div>
-  {% if 'G_CLOUD_7_IS_LIVE' is active_feature %}
-    <aside role="complementary" class="g-cloud-7-notice column-one-third">
-      <h2 id="g-cloud-7-notice-heading">
-        G&#8209;Cloud&nbsp;7 is open for applications
-      </h2>
-      <p class="hint">
-        Deadline for submissions<br />
-        3pm <abbr title="British Summer Time">BST</abbr>, 6 October 2015
-      </p>
-      <p>
-        Register your interest in becoming a G&#8209;Cloud&nbsp;7 supplier.
-      </p>
-      {% if 'G_CLOUD_7_SUPPLIER_GUIDE' is active_feature %}
-        <p class="supplier-info">
-          <a href="">Read G&#8209;Cloud&nbsp;7 information for suppliers</a>
-        </p>
-      {% endif %}
-      <nav role="navigation" aria-labelledby="g-cloud-7-notice-heading">
-        <ul>
-          <li><a href="/suppliers">Log in to existing supplier account</a></li>
-          <li><a href="/suppliers/create">Create new supplier account</a></li>
-        </ul>
-      </nav>
-    </aside>
-    {% else %}
-      <div class="g-cloud-7-notice column-one-third">
+    <div class="framework-message column-one-third">
+      {% if temporary_message %}
+        {# define all variables used by any of the expected templates #}
```

Comment:
I'm not sure this comment is necessary and I'm a bit nervous about comments directly in templates (as apposed to in macro templates) as these files tend to get chopped around a bit and therefore comments can easily be end up separated from the thing they're supposed to be commenting.